src/components/Editor.tsx

```diff
@@ -12,6 +12,11 @@ import { EditorState } from "./SingletonEditor";
 interface ExtendedUnlayerOptions extends UnlayerOptions {
   features: ExtendedFeatures;
   mergeTagsConfig: { sort: boolean };
+  designTagsConfig:
+    | {
+        delimiter: ["[[{", "}]]"];
+      }
+    | any;
```

Comment:
@andresmoschini @maurocardinali el tipado de EmailEditor tiene errores, por eso me costó comprender el porque la configuración de los DesignTags no tenia efecto al cargar el contenido.
Acá les dejo el contexto para que me plantean una mejora de este cambio.

<pre>
<b><i>// Esta es la interfaz que define la configuracion de DesignTag en EmailEditor</i></b>
export interface</b> DesignTagConfig {
    readonly delimeter: [string, string]; <b><i>//<i>Acá <s>delimeter</s> esta mal escrito debe ser delimiter</i></b>
}
</pre>

Mi solución fue agregarle **_any_** al sobrescribir la interfaz

Reply:
🤦‍♂️

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59589

Reply:
Maybe you can wait for the updated version of the definitions, it should be available in few minutes: https://github.com/DefinitelyTyped/DefinitelyTyped#my-pr-is-merged-when-will-the-types-npm-package-be-updated